file/cmd/main_test.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
Should the copyright be updated here?

Reply:
Yes, I have updated the copyright. 
Thank You